frameworks/cassandra/tests/conftest.py

```diff
@@ -0,0 +1,37 @@
+import os
+import json
+import pytest
+import shakedown
+
+
+@pytest.fixture(scope='session', autouse=True)
+def configure_universe(request):
+    stub_urls = {}
+
+    # prepare needed universe repositories
+    stub_universe_urls = os.environ.get('STUB_UNIVERSE_URL')
+    if not stub_universe_urls:
+        return
+    for url in stub_universe_urls.split(' '):
+        package_name = url.split('/')[4] # TODO: does this need to be a random string?
```

Comment:
This is tricky, the string needs to match what is used inside the framework here, right?
https://github.com/mesosphere/dcos-commons/blob/master/frameworks/cassandra/tests/config.py#L11

In such a situation where there needs to be a shared constant and you are using pytest, I generally recommend a simple fixture like:
```
@pytest.fixture
def package_name():
    return 'cassandra'
```
but there may be more ways of doing that here